src/cli-translator/azext_cli_translator/tests/latest/recordings/test_arm_translator.yaml

```diff
@@ -0,0 +1,123 @@
+      {"value": "Dynamic"}, "publicIpAddressSku": {"value": "Basic"}, "virtualMachineName":
+      {"value": "test-vm"}, "virtualMachineComputerName": {"value": "test-vm"}, "virtualMachineRG":
+      {"value": "test-rg"}, "osDiskType": {"value": "Premium_LRS"}, "dataDisks": {"value":
+      [{"lun": 0, "createOption": "attach", "caching": "None", "writeAcceleratorEnabled":
+      false, "id": null, "name": "test-vm_DataDisk_0", "storageAccountType": null,
+      "diskSizeGB": null, "diskEncryptionSet": {"id": null}}]}, "dataDiskResources":
+      {"value": [{"name": "test-vm_DataDisk_0", "sku": "Premium_LRS", "properties":
+      {"diskSizeGB": 16, "creationData": {"createOption": "empty"}}}]}, "virtualMachineSize":
+      {"value": "Standard_D2s_v3"}, "adminUsername": {"value": "AzureUser"}}}}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '6147'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.22.0
+    method: POST
+    uri: https://portal2cli.azurewebsites.net/api/v1
+  response:
+    body:
+      string: '["az account set --subscription 00000000-0000-0000-0000-000000000000","az
+        network vnet create --name test-rg-vnet --resource-group test-rg --address-prefixes
+        10.2.11.0/24 --location japaneast","az network vnet subnet create --name default1
+        --resource-group test-rg --vnet-name test-rg-vnet --address-prefixes 10.2.11.128/25","az
+        network vnet subnet create --name default2 --resource-group test-rg --vnet-name
+        test-rg-vnet --address-prefixes 10.2.11.0/25","az network public-ip create
+        --name test-vm-ip --resource-group test-rg --allocation-method Dynamic --location
+        japaneast --sku Basic","az disk create --name test-vm_DataDisk_0 --resource-group
+        test-rg --location japaneast --size-gb 16 --sku Premium_LRS","az network nic
+        create --name test-vm650 --resource-group test-rg --subnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.Network/virtualNetworks/test-rg-vnet/subnets/default2
+        --location japaneast --public-ip-address /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.Network/publicIpAddresses/test-vm-ip","az
+        vm create --name test-vm --resource-group test-rg --admin-username AzureUser
+        --attach-data-disks /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.Compute/disks/test-vm_DataDisk_0
+        --computer-name test-vm --data-disk-caching 0=None --enable-agent true --image
+        Canonical:UbuntuServer:18.04-LTS:latest --location japaneast --nics /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.Network/networkInterfaces/test-vm650
+        --size Standard_D2s_v3 --storage-sku Premium_LRS\n"]'
```

Comment:
I have fixed the trailing '\n' bug on the service side, please re-record this file.

Reply:
No need to re-record. It will not impact the test.